reference/Client-API.yaml

```diff
@@ -2609,6 +2609,47 @@ paths:
               Example 1:
                 value: eula=true
         description: Write to file
+  '/servers/{server}/files/pull':
+    parameters:
+      - schema:
+          type: string
+        name: server
+        in: path
+        required: true
+    post:
+      summary: ''
```

Comment:
Ideally, fill out the summary to describe what the endpoint is. for instance: "Pull Remote File"